sdk/appconfiguration/tests.yml

```diff
@@ -0,0 +1,27 @@
+trigger: none
+
+jobs:
+  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+    parameters:
+      Matrix:
+        Linux_Python35:
+          OSName: 'Linux'
+          OSVmImage: 'ubuntu-16.04'
+          PythonVersion: '3.5'
+        MacOs_Python37:
+          OSName: 'MacOS'
+          OSVmImage: 'macOS-10.14'
+          PythonVersion: '3.7'
+        Windows_Python27:
+          OSName: 'Windows'
+          OSVmImage: 'vs2017-win2016'
+          PythonVersion: '2.7'
+        Linux_Pypy3:
+          OSName: 'Linux'
+          OSVmImage: 'ubuntu-16.04'
+          PythonVersion: 'pypy3'
+      BuildTargetingString: $(BuildTargetingString)
```

Comment:
This looks good! The variable $(BuildTargetString) is configurable at the pipeline level. You can safely put just your data-plane package name at the pipeline variable level. Can walk you through that if you need!

Reply:
Agreed. Because this tests.yml file is specific to `appconfiguration` it's ok to hardcode the value you want to use here instead of configuring it manually in the pipeline. 

We want to avoid pipelines that require manual configuration because it makes pipeline automation more difficult.

Reply:
Do you mean I can just hard code "azure-data-appconfiguration"?

Reply:
Discussed offline. Set queue time variable on AzureDevOps: `BuildTargetingString` == `"azure-data-appconfiguration"`